lib/srv/regular/proxy.go

```diff
@@ -70,20 +64,20 @@ var ( // failedConnectingToNode counts failed attempts to connect to a node
 			Help:      "Number of SSH connection attempts to a node. Use with `failed_connect_to_node_attempts_total` to get the failure rate.",
 		},
 	)
-
-	prometheusCollectors = []prometheus.Collector{proxiedSessions, failedConnectingToNode, connectingToNode}
 )
 
+func init() {
+	metrics.RegisterPrometheusCollectors(proxiedSessions, failedConnectingToNode, connectingToNode)
```

Comment:
I thought you didn't like doing this in init?

(I actually prefer it this way, FWIW)